### PR TITLE
Task create admin page

### PR DESCRIPTION
**Pull request by katestarks**

### CODE REVIEW

File: php/cms_page.php [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        <form method="post">
+            <label for="addAboutMeText"><h4>Add new information or additional paragraphs here:</h4></label>
+            <textarea class="typeText" name="addAboutMeText" form="addAboutMeText"></textarea>
+            <input type="submit" value="Add text">
+        </form>
+
+        <form method="post">
+            <label for="selectAboutMeText"><h4>Edit text:</h4></label>
+            <select class="aboutMeDropdown" name="selectAboutMeText">
+
```
Comment (Knux2) [inline php/cms_page.php]: Spacing required?

File: php/cms_page.php [hunk 2]
```diff
@@ -0,0 +1,50 @@
+<?php
+//
+//require_once 'dbConnection.php';
+//
+//?>
+
```
Comment (Knux2) [inline php/cms_page.php:6]: (!DOCTYPE html) tag required at top?

File: php/cms_page.php (hunk 1 above)
Comment (Knux2) [inline php/cms_page.php:27]: Space left for dropdown list later?

File: php/cms_page.php [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+            <label for="selectAboutMeText"><h4>Edit text:</h4></label>
+            <select class="aboutMeDropdown" name="selectAboutMeText">
+
+            </select>
+            <input type="submit" value="select text">
+        </form>
+        <form method="post">
+
+            <textarea class="typeText" name="editAboutMeText" form="editAboutMeText">
+
```
Comment (Knux2) [inline php/cms_page.php:33]: Spacing required?

File: php/cms_page.php [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+            <input type="submit" value="Edit text">
+        </form>
+        <form method="post">
+            <label for="deleteAboutMeText"><h4>Select about me text to delete:</h4></label>
+            <select class="aboutMeDropdown" name="deleteAboutMeText">
+
+            </select>
+            <input type="submit" value="Delete">
+        </form>
+
```
Comment (Knux2) [inline php/cms_page.php]: Spacing required?

**Review by Knux2 — COMMENTED** [review]
DOCTYPE at top of php document and spacing in areas of HTML. Everything else, Awesome.

File: php/cms_page.php (hunk 2 above)
Comment (katestarks) [reply php/cms_page.php:6]: Added

File: php/cms_page.php (hunk 1 above)
Comment (katestarks) [reply php/cms_page.php:27]: Left this space as a marker for php

File: php/cms_page.php (hunk 3 above)
Comment (katestarks) [reply php/cms_page.php:33]: Deleted

File: php/cms_page.php (hunk 4 above)
Comment (katestarks) [reply php/cms_page.php]: Deleted

**Review by Knux2 — COMMENTED** [review]
All good

**Pull request merged**